liblttng-ust-java-agent/java/lttng-ust-agent-jul/org/lttng/ust/agent/jul/LttngLogHandler.java

```diff
@@ -84,11 +88,29 @@ public LttngLogHandler() throws IOException, SecurityException {
 			throw new IOException(e);
 		}
 
+		/* Initialize handler specific properties */
+		LogManager manager = LogManager.getLogManager();
+		String cname = getClass().getName();
+		logSource = getBooleanProperty(manager.getProperty(cname + ".logSource"), true);
```

Comment:
I'm guessing `LogManager.getProperty()` will check the contents of the JUL .properties file. It won't check properties passed on the command line with `-D` right? Would it be worth also checking `System.getProperty` for that?